components/stimulation/StimulationStudioWaveformSettingModal.vue

```diff
@@ -450,6 +451,19 @@ export default {
     color_to_display: function () {
       return "background-color: " + this.selected_color;
     },
+    active_duration_in_millis: function () {
```

Comment:
A lot of these calculations are performed already, I think we can just multiply the value on line 703 by .8 since this would replace that part of the check anyway

Reply:
@tannermpeterson yeah I simplified everything, I didn't notice the max_pulse_duration_for_freq